src/xdocs/config_javadoc.xml

```diff
@@ -1704,6 +1704,76 @@ public boolean isSomething()
       </subsection>
     </section>
 
+    <section name="MissingJavadocPackage">
+      <p>Since Checkstyle 8.22</p>
+      <subsection name="Description" id="MissingJavadocPackage_Description">
+        <p>
+            Checks for missing Javadoc comments in package-info.java files.
+        </p>
+
+        <p>
+          Rationale: description and other related documentation for a package can be written up
+          in the package-info.java file and it gets used in the production of the Javadocs.
+          See <a href="https://docs.oracle.com/javase/8/docs/technotes/tools/windows/javadoc.html#packagecomment">
+          link</a> for more info.
+        </p>
+      </subsection>
+
+      <subsection name="Examples" id="MissingJavadocPackage_Examples">
+        <p>
+            To configure the check:
+        </p>
+        <source>
+            &lt;module name="MissingJavadocPackage"/&gt;
+        </source>
+        <p>Example:</p>
+        <source>
+/**
+* Provides API classes
+{@literal *}/
```

Comment:
@pbludov ping
@strkkk `{@literal *}` shouldn't appear here as this was only in Javadoc to prevent thinking it is the end of the javadoc comment.

Reply:
done